main/routes/all_questions_route.py

```diff
@@ -0,0 +1,16 @@
+from .baseRoutes import *
+
+
+@app.route('/api/v1/questions', methods = ['GET'])
+def get_all_questions():     
+    """This endpoint will fetch all questions """
+    listed_questions = []
+    if questions_list == []:
```

Comment:
This can be simply written as 
```python 
if not question_list:
```
Its good to knoe which values are truthy and falsy in any language. You can see the what is what for the case of python in this stackoverflow [thread](https://stackoverflow.com/questions/39983695/what-is-truthy-and-falsy-in-python-how-is-it-different-from-true-and-false)